Assets/Scripts/Configuration/Indicators/Configuration.cs

```diff
@@ -8,10 +8,35 @@
 
 namespace Netherlands3D.Twin.Configuration.Indicators
 {
+    /// <summary>
+    /// The configuration class for indicators.
+    /// </summary>
     [CreateAssetMenu(menuName = "Netherlands3D/Twin/Configuration/Indicators", fileName = "IndicatorConfiguration", order = 0)]
     public class Configuration : ScriptableObject, IConfiguration
     {
-        [SerializeField] private string dossierId;
+        /// <summary>
+        /// The base uri for the API endpoint / URI where to find dossiers or to import dossiers.
+        ///
+        /// This setting can only be set through the configuration file -key:baseUri- and not through query parameters
+        /// to prevent possible abuse.
+        /// </summary>
+        private string baseUri = "https://localhost:7071/api/v1/indicators";
```

Comment:
Zouden we hier iets van een dummy-endpoint voor kunnen aanmaken, zodat men terugkoppeling krijgt dat de setup in essentie werkt, maar deze uri nog vervangen moet worden oid?